doc/readme.html

```diff
@@ -545,6 +545,9 @@ <h4>Optional interfaces to other software packages</h4>
                 <p>
                     <a href="https://github.com/kokkos/kokkos" target="_top">Kokkos</a> implements a programming model in C++ for writing performance portable applications targeting all major HPC platforms.
                     To use a self compiled version, pass <code>-DKOKKOS_DIR=/path/to/kokkos</code> to the deal.II CMake call.
+                    The compiler must be able to compile code for all enabled backends. In case Cuda is enabled in Kokkos and nvcc should be used as device compiler,
+                    it is recommended to use the nvcc_wrapper script that comes with Kokkos as C++ compiler. clang++ or nvc++ (for Kokkos >= 4.0) could be used directly as C++ compiler.
+                    Also, Kokkos must be build with support for device lambdas, e.g., Kokkos_ENABLE_CUDA_LAMBDA=ON when configuring Kokkos with Cuda.
```

Comment:
Add a comment that's is only necessary for Kokkos pre-4.0.

Reply:
[5c7efb3](https://github.com/dealii/dealii/pull/14365/commits/5c7efb35e43a58b7ae3f83700f772cea4f407f2e)